lib/attributes-to-props.js

```diff
@@ -5,9 +5,10 @@ var utilities = require('./utilities');
  * Converts HTML/SVG DOM attributes to React props.
  *
  * @param {object} [attributes={}] - HTML/SVG DOM attributes.
+ * @param {string} [nodeName] - DOM node name. Help's to have smarter behavior while converting DOM attributes to props.
```

Comment:
Could you fix grammar `Helps` or remove the 2nd sentence since `DOM node name.` seems self-explanatory.